CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.1.2 - January 18, 2022
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+#### Bug fixes and improvements
+- Fixed decoding of OpenLR point along line. [5369](https://github.com/mapbox/mapbox-navigation-android/pull/5369)
+- Fixed multi language support in onboard router. [5369](https://github.com/mapbox/mapbox-navigation-android/pull/5369)
+- Fixed native memory leak. [5369](https://github.com/mapbox/mapbox-navigation-android/pull/5369)
+
+### Mapbox dependencies
+This release depends on, and has been tested with, the following Mapbox dependencies:
+- Mapbox Maps SDK `v10.0.2` ([release notes](https://github.com/mapbox/mapbox-maps-android/releases/tag/android-v10.0.2))
+- Mapbox Navigation Native `v79.0.4`
+- Mapbox Core Common `v20.0.0`
```

Comment:
Strange. Previous version of Core Common is `v20.0.3`. Is
`com.mapbox.common:common:${version.mapboxCommonNative}` core common? Or there should be a version of a different lib? 🤔

Reply:
Nah, [it's 20.0.0](https://github.com/mapbox/mapbox-navigation-android/blob/59476c38bae88e0751c8090f0eab161ddc0d0b65/gradle/dependencies.gradle?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L25). Nice shot 🔫  could you fix the line in changelog and release notes

Reply:
Maps SDK in this case brings in the Core Common is `v20.0.3` transitively. We don't see it in this project directly to avoid a mismatch between the version of Core Common that Nav Native and Nav SDK depend on which would make some tests fail. This was specifically done to shorten the critical patch release cycle that was targeting Maps SDK.

Let's leave `v20.0.3` here.

Reply:
You can add a `(transitively)` annotation to the release version to avoid potential confusion.

Reply:
why we didn't update this version for our SDK?